File: Data Structures & Algorithms/sqrtx/submission-4.cs
public class Solution {
    public int MySqrt(int x) {
        int l = 0, r = x;
        int res = 0;

        while (l <= r) {
            int m = l + (r - l) / 2;
            long sq = (long)m * m;

            if (sq > x) {
                r = m - 1;
            } else if (sq < x) {
                l = m + 1;
                res = m;
            } else {
                return m;
            }
        }

        return res;
    }
}

File: Data Structures & Algorithms/length-of-last-word/submission-4.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        res=s.strip()
        print(res)
        n=len(res)-1
        l=0
        for i in res[::-1]:
            if i!=" ":
                l+=1
            else:
                break
        return l

        